docs/UserGuide.md

```diff
@@ -246,7 +246,7 @@ Other examples:
 
 ### 5.3. Listing all existing internship applications
 
-Lists all of your internship applications in WorkBook in a [sorted order](#3-things-to-note).
+Lists all of your internship applications in WorkBook in a [sorted order](#43-behaviour-of-sorted-internship-applications).
```

Comment:
43 == 4.3 isit?

Reply:
yup thats the way the ids are generated for markdown headers